### ISSUE TITLE
Billing Acc does not sync Connect Vs WM

### ISSUE THREAD

**Issue by drasticdpk** [topcoder-platform/work-manager#1061]
Reference  Challenge :
Connect :  https://connect.topcoder.com/projects/24944 
![image](https://user-images.githubusercontent.com/25297859/106454660-353c4d00-64b3-11eb-9c9c-52d15981bc3f.png)

But unable to activate challenge from WM app.
Reference 
![image](https://user-images.githubusercontent.com/25297859/106454748-543adf00-64b3-11eb-9461-89991243a6e2.png)

if this is due to configuration, please enable for a while , I want to test some scenario for copilot payment for Task.
Scenario : Task having payment for copilot too. 
 (Select copilot yourself and add prize too.)

**Comment by drasticdpk** [comment]
cc @maxceem

**Comment by maxceem** [comment]
@rootelement I saw this issue several times when the billing account is there, but it's not picked up by the WM in some reason. Usually, I solve it by manually choosing billing account for the challenge in Direct. Though usually it's not enough, and there is some other validation error which has to be fixed, before I can set billing account in Direct. For example, missing Platform.

**Comment by maxceem** [comment]
@vikasrohit somehow

**Comment by vikasrohit** [comment]
Yep, it is known. And technically this never designed as this. Till now BA has to be set on Direct project, we are working on reading BA from v5 projects (connect) in this shape up project. #1042
